docs/topics/upgrade.md

```diff
@@ -67,89 +67,59 @@ Docs: [clusterctl upgrade plan](https://cluster-api.sigs.k8s.io/clusterctl/comma
-kube-system                           kube-apiserver-host-cluster-control-plane-t97fn                  1/1     Running   560 (33d ago)   79d
-kube-system                           kube-controller-manager-host-cluster-control-plane-t97fn         1/1     Running   492 (33d ago)   79d
-kube-system                           hubble-relay-6676b755f6-l7vcd                                    1/1     Running   0               33d
-kube-system                           hubble-ui-55f87db549-q4bxb                                       2/2     Running   0               33d
-default                               netshoot                                                         1/1     Running   2 (21d ago)     21d
-kube-system                           cilium-l8p7s                                                     1/1     Running   0               20d
-kube-system                           kube-proxy-n7pwh                                                 1/1     Running   0               20d
-kube-system                           kube-scheduler-host-cluster-control-plane-2r25q                  1/1     Running   0               20d
-kube-system                           etcd-host-cluster-control-plane-2r25q                            1/1     Running   0               20d
-kube-system                           kube-apiserver-host-cluster-control-plane-2r25q                  1/1     Running   0               20d
-kube-system                           kube-controller-manager-host-cluster-control-plane-2r25q         1/1     Running   0               20d
-cert-manager                          cert-manager-cainjector-ffb4747bb-bt2l7                          1/1     Running   0               10m
-cert-manager                          cert-manager-99bb69456-ntvz6                                     1/1     Running   0               10m
-cert-manager                          cert-manager-webhook-545bd5d7d8-6cxxk                            1/1     Running   0               10m
-capi-system                           capi-controller-manager-746b4f5db4-zzbv9                         1/1     Running   0               9m17s
-capi-kubeadm-bootstrap-system         capi-kubeadm-bootstrap-controller-manager-8654485994-tpvkf       1/1     Running   0               9m14s
-capi-kubeadm-control-plane-system     capi-kubeadm-control-plane-controller-manager-5d9d9494d5-2mqlc   1/1     Running   0               9m11s
-cluster-api-provider-hetzner-system   caph-controller-manager-566f996fbd-jrqc4                         1/1     Running   0               2m30s
-```
+Check the health of your workload cluster with your preferred tools and ensure that all components are healthy especially apiserver and etcd pods.
 
+After upgrading cluster API, you might be intersted in updating the Kubernetes version of your controlplane and worker nodes. Those details can be found in the [Cluster API documentation](https://cluster-api.sigs.k8s.io/tasks/upgrading-clusters).
```

Comment:
can you add this link to the beginning and specify what we show in this document and what is shown in CAPI's docs?